package.json

```diff
@@ -15,8 +15,9 @@
     "unseed": "npx sequelize-cli db:seed:undo",
     "down": "npx sequelize-cli db:migrate:undo",
     "test": "cross-env NODE_ENV=test jest",
-    "start": "cross-env NODE_ENV=production node server.js",
-    "dev": "cross-env NODE_ENV=development nodemon --experimental-modules --es-module-specifier-resolution=node server.js",
+    "start": "NODE_ENV=development nodemon --exec babel-node src/server.js",
```

Comment:
there is no cross-env on the start script

Reply:
done!